docs/04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enable Kyma with JetStream
+---
+
+This guide shows how to enable JetStream and how it upholds `at least once delivery` quality of service.
+
+### Enable JetStream
+
+Install Kyma and enable the JetStream flag by running:
+
+```
+kyma deploy --value global.jetstream.enabled=true --value global.jetstream.storage=file
+```
+
+> **NOTE:** The storage flag set to `file` enables persistence of messages and streams, in case of NATS server restart.
+
+### At least once delivery guarantees using JetStream backend
+
+1. Create a [function](../../02-get-started/04-trigger-workload-with-event.md#create-a-function), [subscription](../../02-get-started/04-trigger-workload-with-event.md#create-a-subscription) and [trigger the workload with an event](../../02-get-started/04-trigger-workload-with-event.md#trigger-the-workload-with-an-event). Everything should work as usual.
```

Comment:
```suggestion
1. Create a [Function](../../02-get-started/04-trigger-workload-with-event.md#create-a-function), [Subscription](../../02-get-started/04-trigger-workload-with-event.md#create-a-subscription) and [trigger the workload with an event](../../02-get-started/04-trigger-workload-with-event.md#trigger-the-workload-with-an-event).
```
I don't think that the last sentence is required. It should be obvious for the user that everything should work.